test/security.test.js

```diff
@@ -0,0 +1,30 @@
+const expect = require("chai").expect;
```

Comment:
can we follow a naming convention for test file as filename_test.js . So, can we rename this file to security_test.js?

Reply:
Yes.. i will change the file name